client/src/Home.js

```diff
@@ -1,21 +1,159 @@
-import React, { useState, useEffect } from "react";
-import Api from "./Api";
-import BudgetList from "./components/budget/BudgetList";
-function Home() {
-  const [budget, setBudget] = useState([]);
+import React, { useEffect } from "react";
+import { useLocation, useHistory } from "react-router-dom";
+
+import { BulgaLogo, BulgoFooter } from "./shared/logo/Avatars";
+import { Link } from "react-router-dom";
+
+import "./shared/style/Home.css";
+import PigCoin from "./shared/pics/undraw_Savings_re_eq4w (1).png";
+import LadyMoney from "./shared/pics/undraw_investment_data_yfvt.png";
+
+function Home({ setPath }) {
+  const location = useLocation();
+  const history = useHistory();
 
   useEffect(() => {
-    Api.budget.index().then((res) => setBudget(res.data));
+    setPath(location.pathname);
```

Comment:
If we refactor the Header implementation as suggested above, we won't need this setPath hack...